frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-row.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { ArrowCircleDownIcon } from '@patternfly/react-icons';
+import { LoadingInline } from '@console/internal/components/utils';
+
+const prefixedId = (idPrefix, id) => {
+  return idPrefix && id ? `${idPrefix}-${id}` : null;
+};
+
+const BucketsRowTitle = ({ count, title, objectsCount }) => (
+  <div className="co-buckets-card__row-title">
+    <div>{count != null ? `${count} ${title}` : title}</div>
+    <div className="co-buckets-card__row-subtitle">{`${objectsCount} Objects`}</div>
+  </div>
+);
+
+const BucketsItemStatus = ({ error }) => (
+  <div className="co-buckets-card__row-status-item">
+    <ArrowCircleDownIcon />
+    <span className="co-buckets-card__row-status-item-text">{error}</span>
+  </div>
+);
+
+export const BucketsRow = ({ title, bucketsCount, objects, unhealthy }) => (
+  <div id={prefixedId('buckets', title.toLowerCase())} className="co-buckets-card__row">
+    <BucketsRowTitle count={bucketsCount} title={title} objectsCount={objects} />
+    {bucketsCount != null ? (
+      <BucketsItemStatus error={unhealthy} key="status" />
+    ) : (
+      <LoadingInline />
```

Comment:
How do we handle errors? Does this just stay loading forever if there's an error?

Reply:
@spadgett  I am waiting for clarification on loading and unavailable states from UXD. For now while polling data from prometheus there will be `loading component` and in case of null/undefined values a text of 'Unavailable'.
I have updated the screenshots too